biodatasets/medquad/medquad.py

```diff
@@ -0,0 +1,264 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This is an implementation of the dataloader for MedQuAD dataset.
+MedQuAD includes 47,457 medical question-answer pairs created from 12 NIH websites (e.g. cancer.gov, niddk.nih.gov, GARD, MedlinePlus Health Topics). The collection covers 37 question types (e.g. Treatment, Diagnosis, Side Effects) associated with diseases, drugs and other medical entities such as tests.
+We included additional annotations in the XML files, that could be used for diverse IR and NLP tasks, such as the question type, the question focus, its syonyms, its UMLS Concept Unique Identifier (CUI) and Semantic Type.
+We  added the category of the question focus (Disease, Drug or Other) in the 4 MedlinePlus collections. All other collections are about diseases.
+The paper cited below describes the collection, the construction method as well as its use and evaluation within a medical question answering system.
+N.B. We removed the answers from 3 subsets to respect the MedlinePlus copyright (https://medlineplus.gov/copyright.html):
+(1) A.D.A.M. Medical Encyclopedia, (2) MedlinePlus Drug information, and (3) MedlinePlus Herbal medicine and supplement information.
+-- We kept all the other information including the URLs in case you want to crawl the answers. Please contact me if you have any questions.
+
+For more info please visit https://github.com/abachaa/MedQuAD/
+"""
+import json
+import os
+import xml.etree.ElementTree as ET
+from typing import List, Tuple, Dict
+
+import datasets
+import requests
+
+from utils import schemas
+from utils.configs import BigBioConfig
+from utils.constants import Tasks
+
+_CITATION = """\
+@article{BenAbacha-BMC-2019,
+  author    = {Asma {Ben Abacha} and Dina Demner{-}Fushman},
+  title     = {A Question-Entailment Approach to Question Answering},
+  journal   = {{BMC} Bioinform.},
+  volume    = {20},
+  pages     = {511:1--511:23},
+  year      = {2019},
+  url       = {https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3119-4},
+  doi       = {https://doi.org/10.1186/s12859-019-3119-4},
+  biburl    = {https://citation-needed.springer.com/v2/references/10.1186/s12859-019-3119-4?format=refman&flavour=citation},
+}
+"""
+
+_DATASETNAME = "medquad"
+
+_DESCRIPTION = """\
+MedQuAD: Medical Question Answering Dataset  
+MedQuAD includes 47,457 medical question-answer pairs created from 12 NIH websites\
+(e.g. cancer.gov, niddk.nih.gov, GARD, MedlinePlus Health Topics).\
+The collection covers 37 question types (e.g. Treatment, Diagnosis, Side Effects) associated with diseases,\
+drugs and other medical entities such as tests.
+"""
+
+_HOMEPAGE = "https://github.com/abachaa/MedQuAD"
+
+_LICENSE = "https://creativecommons.org/licenses/by/4.0/legalcode"  # TODO: terms aren't available in the repository! In the issue it is 'CC BY 4.0'
```

Comment:
Also, in the description it says `CC BY 4.0` is the license type, so I search the license terms online, I just wanted to make sure this is correct.

Reply:
Where did you find the license of the dataset? I cannot seem to find it...

Reply:
If you check the license field in #169 description, you would see it, although I couldn't find it mentioned in the repo.
I'm not sure, but what I did is assume the license in the desc is correct, I looked for "terms of CC BY 4.0" and pasted the link :)
What do you think it should be @sg-wbi ?